.github/ISSUE_TEMPLATE/first_timers_only.md

```diff
@@ -0,0 +1,57 @@
+---
+name: First Timers Only Issue
+about: Create an issue to welcome a new contributor into the community.
+title: ''
+labels: first-timers-only
+assignees: ''
+
+---
+
+## Background
+
+Overview of your issue here.
+
+## Instructions
+Hi, this is a `first-timer-only` issue. This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.
+
+If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open-source software!
+
+We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝
+
+If you have contributed before, **consider leaving this one for someone new**, and looking through our general [bug](https://github.com/ros-planning/moveit/labels/bug) issues. Thanks!
+
+### 🤔 What you will need to know.
+
+Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.
+
+### 📋 Step by Step
+
+- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
+
+- [ ] 🗄️**Create a local workspace** for making your changes and testing [follwing these instructions](https://moveit.ros.org/install/source/)
+
+- [ ] 📝 **Update** the file(s) `$FILENAME`by applying the change shown in the diff below.
+
+```diff
+$DIFF
+```
+
+- [ ] 🤖 **Apply `clang-format-10`** auto formatting, [following these instructions](https://moveit.ros.org//documentation/contributing/code/?#clang-format-auto-code-formatting)
```

Comment:
This should include clang-tidy, now that I think about it.

Reply:
Good point, except we need to clean up our instructions for clang-tidy.  I couldn't figure out how to get it to work consistently following our instructions recently.  For now, my hope is that those of us who write these `first-timer-only` issues will either make sure the diff doesn't cause clang-tidy issues or help the newcomer with suggestions once it does in their PR.